src/tests/class_tests/smartpeak/executables.cmake

```diff
@@ -15,6 +15,7 @@ set(core_executables_list
 	SequenceSegmentProcessor_test
 	SessionHandler_test
 	Utilities_test
+	Parameters_test
```

Comment:
Are there any additional tests that are missing for `CastValues` as I think you expanded it quite a bit?

Reply:
There is no test for CastValue at all. During the last sprint we planned to do it, but I was about to remove (the day of the sprint, I was trying to do it). In the end no,  and I used it widely instead. so I will reopen the story about testing the CastValue.

Reply:
Let's keep the ones for CastValue then as another PR